config/routes.rb

```diff
@@ -1,58 +1,6 @@
 Rails.application.routes.draw do
-  root 'homes#index'
+  root 'neighborhoods#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.